{"uuid": "f2cd3f90-1abb-4bfb-8b58-b4c06ffe110c", "children": ["4d279fad-72f6-4215-9960-adc3bdd61912"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n", "trace": "  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/test/Desktop/sprint_7/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/test/Desktop/sprint_7/conftest.py\", line 22, in driver\n    driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())\n"}, "start": 1728419172643, "stop": 1728419175326}], "start": 1728419172643, "stop": 1728419175381}